packages/frontend/components/ActionsRow/index.tsx

```diff
@@ -5,8 +5,17 @@ import ActionCard from './ActionCard'
 function ActionsRow() {
   const { actions } = useTreasure()
 
```

Comment:
i realized the scrollbar was cutting off this row in a weird fashion because the cards themselves have a negative margin left on them (-ml-5) which messes up the width computation of ActionsRow....so i manually set the width with computeWidth, instead of letting it be dynamic